.github/workflows/ibis-docs-main.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - main
       - "*.x.x"
+      - "the-epic-split"
```

Comment:
Definitely do not want this here. It will publish docs from all of these branches. While you're at can you remove the `"*.x.x"`.

It's enough to build on PRs.